src/register-org/components/organisation-services-access/organisation-services-access.component.html

```diff
@@ -6,14 +6,12 @@
       <span class="govuk-caption-xl">Register an organisation</span>
       <form [formGroup]="servicesFormGroup">
         <div class="govuk-form-group">
-          <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
-                Which services will your organisation need to access?
-              </h1>
+          <fieldset class="govuk-fieldset">
```

Comment:
@johnbenjamin-hmcts - for accessibility, add `aria-describedby="service-selection-hint"` (once you've added that id on Line 17)

Reply:
Thank you @Daniel-Lam - I have updated the code now.